src/Tools/ExternalAccess/Razor/RazorAnalyzerAssemblyResolver.cs

```diff
@@ -54,5 +55,12 @@ internal static bool TrySetAssemblyResolver(Func<AssemblyName, Assembly?> resolv
 
             return resolver?.Invoke(assemblyName);
         }
+
+        private class AssemblyNameEqualityComparer : IEqualityComparer<AssemblyName>
+        {
+            public bool Equals(AssemblyName? x, AssemblyName? y) => x?.FullName.Equals(y?.FullName) == true;
+
+            public int GetHashCode(AssemblyName obj) => Hash.GetFNVHashCode(obj.FullName);
```

Comment:
I guess this direct call to a hash function is preferable to `string.GetHashCode()`?